docs/user/integration/dynatrace/README.md

```diff
@@ -1,18 +1,19 @@
-# Integrate Kyma with Dynatrace
+# Integrate with Dynatrace
 
 ## Overview
 
-The Kyma Telemetry module supports you in integrating with observability backends in a convenient way. The following example outlines how to integrate with [Dynatrace](https://www.dynatrace.com) as a backend. With Dynatrace, you can get all your monitoring and tracing data into one observability backend to achieve real Application Performance Management with monitoring data in context. Apart from [installing Dynatrace Operator](https://github.com/Dynatrace/dynatrace-operator) and leveraging all of the benefits of Dynatrace, you can also integrate custom metrics and traces with Dynatrace. 
+The Kyma Telemetry module supports you in integrating with observability backends in a convenient way. The following example outlines how to integrate with [Dynatrace](https://www.dynatrace.com) as a backend. With Dynatrace, you can get all your monitoring and tracing data into one observability backend to achieve real Application Performance Management with monitoring data in context. Apart from [installing Dynatrace Operator](https://github.com/Dynatrace/dynatrace-operator) and leveraging all of the benefits of Dynatrace, you can also integrate custom metrics and traces with Dynatrace.
 This tutorial covers only trace and metric ingestion. Log ingestion is not covered.
 
 ![overview](./assets/integration.drawio.svg)
 
-## Prerequisistes 
+## Prerequisistes
 
 - Kyma as the target deployment environment
+- The [Telemetry module](https://kyma-project.io/#/telemetry-manager/user/README) is [installed](https://kyma-project.io/#/02-get-started/08-install-uninstall-upgrade-kyma-module?id=install-uninstall-and-upgrade-kyma-with-a-module)
```

Comment:
We have the conflict with a managed environment where you only enable the module, I'm afraid that people will try to "install" it when using that terminology, what do you think?

Reply:
True. Will adjust that!